it/controllers/bankdetails/HasBankAccountControllerISpec.scala

```diff
@@ -71,6 +71,7 @@ class HasBankAccountControllerISpec extends ControllerISpec {
         .user.isAuthorised()
         .s4l.isEmpty()
         .vatScheme.has("bank-account", Json.toJson(BankAccount(true, None, None, None)))
```

Comment:
minor one. we can remove this stub usage now that we use getSection stub below (just to avoid confusion).